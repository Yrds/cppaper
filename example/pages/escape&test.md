this file name must be escaped on sitemap.xml
