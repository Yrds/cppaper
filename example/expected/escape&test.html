<!DOCTYPE html>
<html>
  <head>
    <meta charset="UTF-8" />
    <meta name="viewport" content="width=device-width" />
    <title>Yuri Santos - escape&test</title>
  </head>
  <body>
    <p>this file name must be escaped on sitemap.xml</p>

  </body>
</html>
